src/arch/riscv64/kernel/mod.rs

```diff
@@ -199,10 +202,8 @@ pub fn boot_next_processor() {
 			next_hart_id
 		);
 
-		// TODO: Old: Changing cpu_online will cause uhyve to start the next processor
 		CPU_ONLINE.fetch_add(1, Ordering::Release);
 
-		//When running bare-metal/QEMU we use the firmware to start the next hart
```

Comment:
I think these comments should be kept. The lower one is still true, and the upper one is still to-do, since Uhyve does not support RISC-V yet.